### ISSUE TITLE
Front page collapsable graph

### ISSUE THREAD

**Issue by DomitillaRaimondo** [rukayaj/species#55]
For Rukaya what Elizma and I are wondering is if it would be possible to make each bar on the graph link with the image below and if when the screen is reduced (e.g. when looking at the site from a cellphone) it would be possible for the graph to collapse right down to having just each taxon bar and image appear below one another. 

Contact me if it is not.

**Comment by rukayaj** [comment]
Yes I know exactly what you mean. I am not sure how possible it is as the graph is drawn onto the page as 1 object, but I will take a look tomorrow.

**Comment by DomitillaRaimondo** [comment]
☺
I like your github setup Dewidine told me to put the tasks there.

**Comment by rukayaj** [comment]
Yes it's great for keeping track of why we decided to do certain things, what to prioritise, etc. And also then @fhatani23 and I can keep track of who is doing what and so on.

**Comment by rukayaj** [comment]
Note to myself - this should also include each individual image linking to the taxonomic tree node relevant for them. That bit should be quite easy to do once the rest is done.
